src/applications/simple-forms/21-0966/config/form.js

```diff
@@ -8,25 +8,27 @@ import ConfirmationPage from '../containers/ConfirmationPage';
 import getHelp from '../../shared/components/GetFormHelp';
 import preparerIdentification from '../pages/preparerIdentification';
 import veteranBenefitSelection from '../pages/veteranBenefitSelection';
-import preparerPersonalInformation from '../pages/preparerPersonalInformation';
-import preparerIdentificationInformation from '../pages/preparerIdentificationInformation';
-import mailingAddress from '../pages/mailingAddress';
-import phoneAndEmailAddress from '../pages/phoneAndEmailAddress';
+import survivingDependentPersonalInformation from '../pages/survivingDependentPersonalInformation';
+import survivingDependentIdentificationInformation from '../pages/survivingDependentIdentificationInformation';
+import survivingDependentMailingAddress from '../pages/survivingDependentMailingAddress';
+import survivingDependentPhoneAndEmailAddress from '../pages/survivingDependentPhoneAndEmailAddress';
```

Comment:
Thanks for renaming these. yeah, in retrospect, it's much better to be specific like this. 😄